src/applications/personalization/profile-2/components/Profile2Wrapper.jsx

```diff
@@ -23,14 +23,27 @@ const Profile2 = ({ children, routes }) => {
   const onPersonalInformationMobile =
     activeLocation === '/profile/personal-information' && !isWideScreen();
 
+  // Without a verified identity, we want to show 'Home - Account Security'
+  const showLOA1BreadCrumb =
+    (!isLOA3 || !isInMVI) && activeLocation === '/profile/account-security';
```

Comment:
rats. it looks like we don't have a simple way to get `/profile/account-security` from the `routes` object.

Reply:
Yeah, I noticed that. I don't think it's necessary at this point to abstract those routes further into constant files either.